coordinator/src/admin.rs

```diff
@@ -339,36 +331,10 @@ pub async fn collaborative_revert(
 
 #[instrument(skip_all, err(Debug))]
 pub async fn expert_collaborative_revert(
-    State(state): State<Arc<AppState>>,
-    revert_params: Json<CollaborativeRevertCoordinatorExpertRequest>,
+    State(_state): State<Arc<AppState>>,
+    _revert_params: Json<CollaborativeRevertCoordinatorExpertRequest>,
 ) -> Result<(), AppError> {
-    let channel_id_hex = revert_params.channel_id.clone();
-    let channel_id = parse_channel_id(channel_id_hex.as_str())
-        .map_err(|e| AppError::BadRequest(format!("Invalid channel ID provided: {e:#}")))?;
-
-    let funding_txo = OutPoint {
-        txid: revert_params.txid,
-        vout: revert_params.vout,
-    };
-
-    collaborative_revert::propose_collaborative_revert_without_channel_details(
-        state.node.inner.clone(),
-        state.pool.clone(),
-        state.auth_users_notifier.clone(),
-        channel_id,
-        funding_txo,
-        revert_params.coordinator_amount,
-        revert_params.fee_rate_sats_vb,
-        revert_params.price,
-    )
-    .await
-    .map_err(|e| {
-        AppError::InternalServerError(format!("Could not collaboratively revert channel: {e:#}"))
-    })?;
-
-    tracing::info!(channel_id = channel_id_hex, "Proposed collaborative revert");
-
-    Ok(())
+    unimplemented!("This needs to be reimplemented on top of dlc-channels");
```

Comment:
❓ Do we still need an "expert" collab revert option? Now that we have moved away from ldk, we do not run into the issue that the lightning channel might have been gone, which required us to implement an alternative collab revert in the first place.

Reply:
I can't say I fully understand all states of DLC channel yet, but I believe my solution only works if the DlcChannel is in `Signed`. Maybe all DlcChannels are always `Signed`?

Reply:
No, not all dlc channels are `Signed`. But channels which are not signed should either mean the channel is not on-chain yet or (about to go) on-chain already.

But fair enough, lets look at it some other time then.

Reply:
I've removed this function for now